src/index.js

```diff
@@ -43,6 +45,11 @@ import { ReapplySelected } from './scripts/reapply-selected';
     extraTriggersOnActions: {
       cancel: 'cancelKYFBForm',
     },
+    submissionInfo: {
+      projectID: '5f44969319d1a97a819d80a7',
```

Comment:
should we be using the internal id's instead of the nice alias names for form.io?